cdap-ui/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "cdap-ui",
-  "version": "0.1.3",
+  "version": "0.2.8",
```

Comment:
cdap-master is using 2.8.0, @sreevatsanraman should this be the same as cdap version?